dask/Dockerfile

```diff
@@ -1,28 +1,27 @@
-ARG CUDA_VER=11.2
-ARG LINUX_VER=ubuntu18.04
+ARG CUDA_VER=11.5.2
+ARG LINUX_VER=ubuntu20.04
+ARG PYTHON_VER=3.9
 
-FROM gpuci/miniforge-cuda:$CUDA_VER-devel-$LINUX_VER
+FROM rapidsai/miniforge-cuda:cuda$CUDA_VER-base-$LINUX_VER-py$PYTHON_VER
 
-ARG CUDA_VER=11.2
+ARG CUDA_VER=11.5.2
 ARG PYTHON_VER=3.9
-ARG NUMPY_VER=1.20.1
-ARG RAPIDS_VER=21.08
-ARG UCX_PY_VER=0.21
+ARG RAPIDS_VER=23.12
+ARG UCX_PY_VER=0.35
 
 COPY environment.yml /rapids.yml
 ADD https://raw.githubusercontent.com/dask/dask/main/continuous_integration/environment-$PYTHON_VER.yaml /dask.yml
 
 RUN conda config --set ssl_verify false
 
-RUN conda install -c gpuci gpuci-tools
+RUN mamba install -c gpuci gpuci-tools
 
-RUN gpuci_conda_retry install -c conda-forge mamba conda-merge "conda>=4.14"
+RUN gpuci_mamba_retry install conda-merge git
```

Comment:
anything related to `gpuci` has been deprecated.

We have some similar tools below that we use for the RAPIDS CI images that you could install if you want:

- https://github.com/rapidsai/gha-tools

The `gpuci_*` commands have all been superseded by `rapids-*` commands.